src/qr.rs

```diff
@@ -120,6 +125,15 @@ pub enum Qr {
         ticket: iroh::provider::Ticket,
     },
 
+    /// Provides a backup that can be retrieved using new iroh-net based protocol.
```

Comment:
I suggest finding a better name for this.  "new iroh-net based protocol" sounds fine right now, but in 6 months it won't.